tests/test_database.py

```diff
@@ -0,0 +1,279 @@
+import os
```

Comment:
I assume that the file header is missing to be consistent with the rest of the files? I think we should consider adding the header to this new file and maybe all other files too.

```suggestion
"""
    PyHSS Database test
    Copyright (C) 2025 sysmocom - s.f.m.c. GmbH <info@sysmocom.de>

    SPDX-License-Identifier: AGPL-3.0-or-later

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU Affero General Public License as published
    by the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU Affero General Public License for more details.

    You should have received a copy of the GNU Affero General Public License
    along with this program.  If not, see <https://www.gnu.org/licenses/>.
"""
import os
```